notify/notify.go

```diff
@@ -284,6 +285,11 @@ func NewMetrics(r prometheus.Registerer, ff featurecontrol.Flagger) *Metrics {
 			Name:      "notification_requests_failed_total",
 			Help:      "The total number of failed notification requests.",
 		}, labels),
+		numAlertsSuppressedTotal: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: "alertmanager",
+			Name:      "alerts_supressed_total",
```

Comment:
```suggestion
			Name:      "notifications_supressed_total",
```

Reply:
I'll fix this with the rest of the feedback (and I'll fix my typo there too :wink: )